glossary.yml

```diff
@@ -2855,6 +2855,22 @@
       not their data types or units).
 
 
+- slug: help_system
+  ref:
+    - documentation
+    - help_command
+    - help_function
+  en:
+    term: "help (command)"
+    def: >
+      Opens a help page for the command. The system provides detailed information
+      on how the command works and how it is used.
```

Comment:
I'd like to see this a bit more general; in programming language interpreters help is usually a command, but for things that run via the command line, it is an option handled by the parser. It also seems a bit weird to have the slug be `help_system`, but the term be `help (command)`, particularly when `help_command` is then listed as one of the references.

Reply:
I agree that a broader definition would be helpful. I wrote this in the context of the R lessons and as also described for [Help (command)](https://en.wikipedia.org/wiki/Help_(command)) here.

Rather than changing this, I propose to change the slug to `help_command` and leave room for a wider "help (system)" definition.

Reply:
Addressed by ae6d55784bde84d091b4d90eee71f474ce5bfe31